source/GameInterface/Services/Clans/Patches/ClanAddRenownPatch.cs

```diff
@@ -16,6 +16,8 @@ public class ClanAddRenownPatch
 
         static bool Prefix(ref Clan __instance, float value, bool shouldNotify = true)
         {
+            if (value == 0f) return false;
```

Comment:
move `if (AllowedInstance.IsAllowed(clan)) return true;` after this line. if we want to allow this call then it should be one of the first checks